### PR TITLE
tasks were unified, Grunt = Gulp :)

### PR DESCRIPTION
**Pull request by arzafran**
Unify Grunt and gulp tasks, added grunt zip, also added browsersync to grunt instead of connect.

BTW remover concurrent, since it was the cause of all evil.

@andresmoschini @joseguti84 @damianmuti @ggmanuel @nmartello-makingsense @matiasbeckerle @fcoronelmakingsense @lcalvino @nazarenooviedo

### CODE REVIEW

**Comment by damianmuti** [comment]
For some reason, the watch task for icon-fonts, html, images and JS files is not working (tested all).

Besides that, a couple of questions:
- Why are we using cssmin instead of cssnano?
- Is there any specific reason for why imagemin is configured? We have kraken for that.

**Comment by arzafran** [comment]
Regarding the comments:
Cssmin uses css-clean, which turned out to be the best in minifying, and it's the same that we're using in gulp.
Imagemin works better than any other tool to compress images, and instead of using a paid online service it minifies the files so you can preview the images in Dev environment.

Last but not least, I'll take a look on the watch tasks, maybe there's a typo, since I had to run them separately in the run task  so grunt can watch sass for changes and build the docs.

File: Gruntfile.js [hunk 1]
```diff
@@ -31,28 +28,41 @@ module.exports = function(grunt) {
...
-    connect: {
-      server: {
+    browserSync: {
+      dev: {
+        bsFiles: {
+          src: [
+            '<%= config.folder_dev %>/css/*.css',
+            '<%= config.folder_dev %>/img/*',
+            '<%= config.folder_dev %>/js/*.js',
+            '<%= config.folder_dev %>/js/*.map',
```
Comment (matiasbeckerle) [inline Gruntfile.js]: Maybe you can remove this line and put `<%= config.folder_dev %>/js/*` in the one above.

File: Gruntfile.js [hunk 2]
```diff
@@ -31,28 +28,41 @@ module.exports = function(grunt) {
...
-          livereload: true,
-          debug: true
+          server: {
+            baseDir: '<%= config.folder_doc %>'
+          }
         }
       }
     },
 
-
```
Comment (matiasbeckerle) [inline Gruntfile.js]: I just love you for removing empty lines.

File: Gruntfile.js [hunk 3]
```diff
@@ -192,110 +209,37 @@ module.exports = function(grunt) {
...
       },
-      dynamic: {
-        files: [{
-          expand: true,
-          cwd: '<%= config.folder_dist %>/img/',
-          src: ['**/*.{png,jpg,jpeg,gif}'],
-          dest: '<%= config.folder_dist %>/img/'
-        }]
+      files: {
+        src: ['<%= config.folder_assets %>/js/vendor/*.js', '<%= config.folder_assets %>/js/*.js'],
```
Comment (matiasbeckerle) [inline Gruntfile.js]: Minor: through all the changes you have been using multiple lines when comes to JS arrays but here you use one line. Why? IMO this is the best example of a nice scenario where it is easier to identify what the source code does putting in multiple lines.

TL;DR: I would rather prefer to use multiple lines in arrays like this one and avoid it on simple strings like `['apple', 'oranges']`.

File: Gruntfile.js [hunk 4]
```diff
@@ -327,65 +271,60 @@ module.exports = function(grunt) {
           },
         }
       }
+    },
+
+    compress: {
+      build: {
+        options: {
+          archive: 'deploy --' + grunt.template.today('yyyy-mm-dd') + '.zip',
```
Comment (matiasbeckerle) [inline Gruntfile.js]: A space in the filename? Are you sure?

Comment (matiasbeckerle) [inline Gruntfile.js]: Not sure how frequents are this task run but I suggest you to add the HH:mm:ss too.

File: package.json [hunk 5]
```diff
@@ -57,14 +57,12 @@
     "del": "~2.2.0",
     "grunt": "~0.4.5",
     "grunt-bowercopy": "~1.2.0",
-    "grunt-concurrent": "~1.0.0",
-    "grunt-contrib-connect": "~0.9.0",
-    "grunt-contrib-copy": "~0.7.0",
-    "grunt-contrib-kraken": "~0.1.3",
+    "grunt-browser-sync": "^2.2.0",
```
Comment (matiasbeckerle) [inline package.json]: I suggest you to stick with the `~` instead of `^`.

File: Gruntfile.js (hunk 4 above)
Comment (arzafran) [inline Gruntfile.js]: Nice catch!

File: Gruntfile.js (hunk 2 above)
Comment (arzafran) [inline Gruntfile.js]: Beauty is in the eye of the beholder.

File: Gruntfile.js (hunk 3 above)
Comment (arzafran) [inline Gruntfile.js]: will do!
